querybook/webapp/ui/ViewersList/ViewersList.tsx

```diff
@@ -0,0 +1,144 @@
+import clsx from 'clsx';
+import * as React from 'react';
+
+import { AccessRequestButton } from 'components/AccessRequestButton/AccessRequestButton';
+import { AccessRequestPermissionPicker } from 'components/AccessRequestPermissionPicker/AccessRequestPermissionPicker';
+import { ViewerPermissionPicker } from 'components/DataDocViewersList/ViewerPermissionPicker';
+import { UserBadge } from 'components/UserBadge/UserBadge';
+import { UserSelect } from 'components/UserSelect/UserSelect';
+import { IAccessRequest } from 'const/accessRequest';
+import { IViewerInfo, Permission } from 'lib/data-doc/datadoc-permission';
+import { StyledText } from 'ui/StyledText/StyledText';
+import { Tabs } from 'ui/Tabs/Tabs';
+
+import './ViewersList.scss';
+
+interface IProps {
+    className?: string;
+    entityType: string;
```

Comment:
i think entityName would be more clear since this prop is only used for display reasons